enterprise/backend/test/metabase_enterprise/snippet_collections/api/native_query_snippet_test.clj

```diff
@@ -155,3 +155,23 @@
                [{:id (:id snippet) :name "My Snippet"}
                 {:id (:id sub-collection) :name "Nested Snippet Collection"}]
                (:data (mt/user-http-request :rasta :get 200 (format "collection/%d/items" (:id collection)))))))))))
+
+(deftest ee-disabled-snippets-graph-test
```

Comment:
Moved from `enterprise/backend/test/metabase_enterprise/enhancements/api/collection_test.clj`, the old location made no sense at all since it was testing an endpoint in this module